src/foam/u2/grid/testdata/HeroModels.js

```diff
@@ -0,0 +1,213 @@
+foam.CLASS({
+    refines: 'foam.core.Property',
+    properties: [
+        {
+           name: 'referenceDAOKey',
+           documentation: 'get the property object from property Id through DAO', 
+        }, 
+        {
+            class: 'Function',
+            name: 'validator',
```

Comment:
I pulled newest foam, grepped for validation and there's none. is validation been removed?